plugin/caffe/caffe_blob.h

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) 2016 by Contributors
+ * \file caffe_blob.h
+ * \brief conversion between tensor and caffeBlob
+ * \author Haoran Wang 
+*/
+#ifndef PLUGIN_CAFFE_CAFFE_BLOB_H_
+#define PLUGIN_CAFFE_CAFFE_BLOB_H_
+
+#include<caffe/blob.hpp>
+#include<mshadow/tensor.h>
+#include<mshadow/tensor_blob.h>
+#include<vector>
+
+namespace mxnet {
+namespace op {
+
+using mshadow::TBlob;
```

Comment:
Don't put global using in headers under common namespace. Consider moving them into functions or mxnet/op/caffe namespace